test/units/TEST-74-AUX-UTILS.ssh.sh

```diff
@@ -43,8 +43,13 @@ usermod -U root
 
 mkdir -p /etc/ssh
 test -f /etc/ssh/ssh_host_rsa_key || ssh-keygen -t rsa -C '' -N '' -f /etc/ssh/ssh_host_rsa_key
-echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
-echo "LogLevel DEBUG3" >> /etc/ssh/sshd_config
+
+# Use /etc/ssh/sshd_config.d/ to not overwrite sshd config when config is in
+# /usr/etc/ssh/sshd_config
```

Comment:
typo in the path?

Reply:
Not sure, maybe the comment isn't well write, but path seems correct. 
" If `/usr/etc/ssh/sshd_config` it is better to use `/etc/ssh/sshd_config.d/` to not overwrite sshd config from vendor directory"

Reply:
On Tumbleweed it is, with the move from /etc to /usr/etc. At least I have this file in my VMs, and in the VM created by mkosi using Tumbleweed image. But of course I will remove the comment :)